FastSurferCNN/utils/arg_types.py

```diff
@@ -22,22 +22,25 @@
 
 
 def vox_size(a: str) -> VoxSizeOption:
-    """Convert the vox_size argument to 'min' or a valid voxel size.
+    """
+    Convert the vox_size argument to 'min' or a valid voxel size.
 
     Parameters
     ----------
     a : str
-        vox size type. Can be auto, bin or a number between 1 an 0
+        Vox size type. Can be auto, bin or a number between 1 an 0.
 
     Returns
     -------
-    [MISSING]
+    str or float
+        If 'auto' or 'min' is provided, it returns a string('auto' or 'min').
+        If a valid voxel size (between 0 and 1) is provided, it returns a float.
+
 
     Raises
     ------
     argparse.ArgumentTypeError
         An error from creating or using an argument. Additionally, vox_sizes may be 'min'.
```

Comment:
Should describe, when an error is raised (=if the arguemnt is not "min", "auto" or convertible to a float between 0 and 1)